CONTRIBUTING.md

```diff
@@ -2,54 +2,65 @@
-3. [log into the projects portal](https://projects.eclipse.org/) and look for
-   ["Eclipse Contributor Agreement"](https://www.eclipse.org/legal/ECA.php), and
-4. ensure that you _sign-off_ your Git commits, and
-5. ensure that you use the _same_ email address as your Eclipse Foundation in
-   commits.
+1. [Create an account at Eclipse](https://dev.eclipse.org/site_login/)
+2. Add your GitHub user name in your account settings
+3. [Log into the projects portal](https://projects.eclipse.org/), look for
+   ["Eclipse Contributor Agreement"](https://www.eclipse.org/legal/ECA.php), and agree to the terms.
+4. Ensure that you _sign-off_ your Git commits with the _same_ email address as your Eclipse Foundation profile.
 
 ## Technical considerations
 
-Please make sure your code compiles by running `mvn clean verify` which will
-execute both unit and integration test phases.  Additionally, consider using 
-http://travis-ci.org to validate your branches before you even put them into
-pull requests.  All pull requests will be validated by Travis-ci in any case
+1. Please make sure your code compiles by running `mvn clean verify` which will
+execute both unit and integration test phases.  Additionally, consider using
+[Travis CI](http://travis-ci.org) to validate your branches before you even put them into
+pull requests.  All pull requests will be validated by Travis CI in any case
 and must pass before being merged.
-
-Again, check that your author email in commits is the same as your GitHub account, 
+2. When commiting, your author email must be the same as your GitHub account,
 and make sure that you sign-off every commit (`git commit -s`).
-
-Do not make pull requests from your `master` branch, please use topic branches
+3. Do not make pull requests from your `master` branch, please use topic branches
 instead.
-
-When submitting code, please make every effort to follow existing conventions
+4. When submitting code, please make every effort to follow existing conventions
 and style in order to keep the code as readable as possible.
-
-Please provide meaningful commit messages.
-
-Do not forget to mention the related issue, if any.
-
-Finally, a contribution is not a good contribution unless it comes with unit
+5. Please provide meaningful commit messages.
+6. Do not forget to mention the related issue, if any.
+7. A contribution is not a good contribution unless it comes with unit
 tests, integration tests and documentation.
 
 ## Coding Style
 
-Please, consider Coding Style for the different languages used in the codebase as 
+Please, consider Coding Style for the different languages used in the codebase as
 described here: https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style
+
+## Development process
+
+This project is separated into many smaller projects (git subprojects).
+Those subprojects can be found at the [dirigiblelabs](https://github.com/dirigiblelabs) GitHub page.
```

Comment:
Those subprojects -> Some of the modules are standard Java Maven modules and some of them are following the WebJars module structure. The WebJars ones

Reply:
Added the information.